daiquiri/__init__.py

```diff
@@ -88,8 +88,27 @@ def setup(level=logging.WARNING, outputs=[output.STDERR], program_name=None,
         root_logger.removeHandler(handler)
 
     # Add configured handlers
-    for o in outputs:
-        o.add_to_logger(root_logger)
+    for out in outputs:
+        # instantiating an output from a dict of options
+        if isinstance(out, dict):
```

Comment:
Ok, then leave it out of this PR. If anybody comes with good argument about that, we can revisit later in a new PR!